x-pack/plugins/ml/public/application/data_frame_analytics/pages/job_map/components/controls.tsx

```diff
@@ -94,27 +150,91 @@ export const Controls: FC<Props> = ({ analyticsId, modelId, details, getNodeData
     };
   }, [cy, deselect]);
 
+  useEffect(() => {
+    // Update elements and close the flyout
```

Comment:
nit: Instead of the comment you can define a callback with a named function instead of anonymous, e.g. `updateElementsOnClose`

Reply:
Updated in 626ab62bb02d68546b25cf3cbf527a0fd88cc1a9

Reply:
https://github.com/elastic/kibana/pull/84299#discussion_r535929361